webpack.common.js

```diff
@@ -35,12 +93,75 @@ module.exports = {
           {
             loader: 'sass-loader',
           },
-          // {
-          //   loader: 'postcss-loader',
-          // },
         ],
       },
     ],
   },
-  plugins: [new CleanWebpackPlugin()],
+  plugins: [
+    new CleanWebpackPlugin(),
+    new HtmlWebpackPlugin({
+      filename: resolve(pythonPath, 'webpack-macros.html'),
+      inject: false,
+      minify: false,
+      css: true,
+      templateContent: macroTemplate
+    }),
+    new CopyPlugin([
+      // fonts
+      {
+        context: './node_modules/@fortawesome/fontawesome-free',
+        from: 'LICENSE.txt',
+        to: resolve(vendorPaths.fontAwesome, 'LICENSE.txt')
+      },
+      {
+        context: './node_modules/@fortawesome/fontawesome-free/css',
+        from: 'all.min.css',
+        to: resolve(vendorPaths.fontAwesome, 'css')
+      },
+      {
+        context: './node_modules/@fortawesome/fontawesome-free',
+        from: 'webfonts',
+        to: resolve(vendorPaths.fontAwesome, 'webfonts')
+      },
+      // opensans
+      {
+        context: './node_modules/@openfonts/open-sans_all',
+        from: 'files/*-400*',
+        flatten: true,
+        to: resolve(vendorPaths.openSans, 'files')
+      },
+      {
+        context: './node_modules/@openfonts/open-sans_all',
+        from: 'LICENSE.md',
+        to: resolve(vendorPaths.openSans, 'LICENSE.md')
+      },
+      {
+        context: './node_modules/@openfonts/open-sans_all',
+        from: 'index.css',
+        to: resolve(vendorPaths.openSans, 'index.css')
+      },
+      // lato
+      {
+        context: './node_modules/@openfonts/lato_latin-ext',
+        from: 'files/*-400*',
+        flatten: true,
+        to: resolve(vendorPaths.lato, 'files')
+      },
+      {
+        context: './node_modules/@openfonts/lato_latin-ext',
+        from: 'LICENSE.md',
+        to: resolve(vendorPaths.lato, 'LICENSE.md')
+      },
+      {
+        context: './node_modules/@openfonts/lato_latin-ext',
+        from: 'index.css',
+        to: resolve(vendorPaths.lato, 'index.css')
+      },
+    ]),
+    new webpack.ProvidePlugin({
+      '$': "jquery",
+      'jQuery': "jquery",
+      'Popper': 'popper.js'
```

Comment:
What is this change doing?

Reply:
this is one of those hangover things from trying to pack bootstrap... and may conflict with the `externals`. we could experiment with removing.

Reply:
removed this in d76ef111... doesn't seem to break anything.